release/release.go

```diff
@@ -151,6 +151,38 @@ func CheckUpstreamRelease(ctx context.Context, client *github.Client, org, repo
 	return releases, nil
 }
 
+func KubernetesGoVersion(ctx context.Context, client *github.Client, version string) (string, error) {
+	const (
+		owner string = "kubernetes"
+		repo  string = "kubernetes"
+	)
+
+	file, _, _, err := client.Repositories.GetContents(ctx, owner, repo, ".go-version", &github.RepositoryContentGetOptions{
+		Ref: version,
+	})
+
```

Comment:
```suggestion
```

Reply:
Done

Reply:
Yeah, my  Vscode lint not worked as expected, now I removed this blank lines :)